packages/zoe/test/swingsetTests/zoe-metering/test-zoe-metering.js

```diff
@@ -36,7 +36,7 @@ const infiniteInstallLoopLog = [
   'installing infiniteInstallLoop',
   'error: RangeError: Compute meter exceeded',
 ];
-test('zoe - metering - infinite loop in installation', async t => {
+test.skip('zoe - metering - infinite loop in installation', async t => {
```

Comment:
might add a comment that these are waiting on https://github.com/Agoric/SES-shim/issues/377 to be fixed and published and upgraded-to

Reply:
Yep.